### DISCUSSION TITLE
Opencanary source IP from webhook

### DISCUSSION THREAD

**Discussion by chrisreeves-** [thinkst/opencanary#315] [General]
Hey!

I'm trying to lock down the source IP on a firewall to only allow certain IP's from the Opencanary webhook notifier. Is it just `52.18.63.80` from AWS or are there others/a range?

TIA

**Comment by SecuriLee** [comment]
I use webhooks to report from OC.  They come from the host itself but I avoid firewall rules, I actually have my OCs and my Splunk also in a Tailnet using Tailscale.

If you're running your OC instance in AWS on that IP, you would be whitelisting TCP/port for that IP to your logging solution.

**Comment by jayjb** [comment]
Hi @chrisreeves- 
I've moved this to issues and ill respond there.
